### PR TITLE
todo redux

### PR DESCRIPTION
**Pull request by donizer**
[DEMO LINK](https://donizer.github.io/react_redux-list-of-todos/)

### CODE REVIEW

File: src/components/TodoFilter/TodoFilter.tsx [hunk 1]
```diff
@@ -1,17 +1,74 @@
...
+  };
+
+  const clearQuery = () => {
+    dispatch(filterActions.clearQuery());
+  };
+
+  const getfilteredTodos = (): Todo[] => {
+    let filteredTodos = [...allTodos];
+
+    if (status === 'completed') {
```
Comment (lerastarynets) [inline src/components/TodoFilter/TodoFilter.tsx:34]: why not using values from Statuses enum here? and better use switch/case here

File: src/components/TodoModal/TodoModal.tsx [hunk 2]
```diff
@@ -1,40 +1,82 @@
...
-            <strong className="has-text-success">Done</strong>
-            {' by '}
-            <a href="mailto:Sincere@april.biz">Leanne Graham</a>
-          </p>
+            <p className="block" data-cy="modal-user">
+              {!todo.completed ? (
+                <strong className="has-text-danger">Planned</strong>
+              ) : (
+                <strong className="has-text-success">Done</strong>
+              )}
```
Comment (lerastarynets) [inline src/components/TodoModal/TodoModal.tsx:73]:
```suggestion
                <strong className=`has-text-{todo.completed  ? 'success' : 'danger'}`>{todo.completed  ? 'Done' : 'Planned'}</strong>
```

**Review by lerastarynets — CHANGES_REQUESTED** [review]
I see you did a hard work) Let's improve it a little bit

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Looks good

**Pull request closed without merging**